Examples/restcomm-olympus-nopod/restcomm-olympus/Utils.m

```diff
@@ -40,11 +39,21 @@
 NSString *const kTurnCanidateTimeoutKey = @"turn-candidate-timeout";
 NSString *const kIsFirstTimeKey = @"is-first-time";
 NSString *const kPendingInterappKey = @"pending-interapp-uri";
-NSString *const kSignalingCertificateKey =@"signaling-certificate-dir";
+NSString *const kSignalingCertificateKey = @"signaling-certificate-dir";
+
+NSString *const kPushAccountKey = @"push-account";
```

Comment:
The same keys are used in RCDevice, right? If so I think they should be moved in RCDevice so that they can be referenced by any client App.

Reply:
Ok.